src/device/pf_cmdev.c

```diff
@@ -4869,7 +4869,8 @@ int pf_cmdev_rm_dcontrol_ind (
    pnet_t * net,
    pf_ar_t * p_ar,
    pf_control_block_t * p_control_io,
-   pnet_result_t * p_dcontrol_result)
+   pnet_result_t * p_dcontrol_result,
+   bool * p_set_state_paramend)
```

Comment:
paramend too similar to PRMEND? use param_end or parameter_end or prmend

Reply:
Now `p_set_state_prmend`
Done